Resources/Prototypes/Catalog/Cargo/cargo_food.yml

```diff
@@ -21,3 +21,16 @@
   cost: 4000
   category: Food
   group: market
+
+- type: cargoProduct
+  name: "kitchen supplies crate"
+  id: FoodCook
+  description: Extra kitchen supplies, in case the botanists are absent.
+  icon:
+    sprite: Objects/Consumable/Food/ingredients.rsi
+    state: flour-big
+  product: CrateFoodCooking
+  cost: 1000
```

Comment:
this should be considerably more expensive imo. at least 5-6000?

Reply:
You absolutely right. I have underestimated nutriment values of different types of food. I'll set crate cost at 5000. However, I think that it can be reduced even to 4000, to make it more attractive in competition with 4 (5) crates of emergency pizza.